app/src/main/res/values/preferences.xml

```diff
@@ -7,42 +7,101 @@
     <!-- Category general -->
     <string name="preference_key_category_general" translatable="false">preference_key_category_general</string>
 
+    <!-- Automatic schedule updates -->
```

Comment:
Why mix constants and translatable UI strings in one file?
Why do UI strings for settings live in a separate file? Wouldn't a single file be much easier for translators?

Reply:
Thanks. I guess you referred to `preference_title_app_notification_settings`. I moved the strings in bc6bbd636d1a609e46d9cd17b6b8a9f8f9528d1c.

Reply:
I want to have all preference resources in a single place. The mixed location in the past turned out to introduce a lot of errors.